src/boot/efi/boot.c

```diff
@@ -69,6 +69,8 @@ typedef struct {
         CHAR16 *entry_oneshot;
         CHAR16 *options_edit;
         BOOLEAN no_editor;
+        BOOLEAN no_auto_entries;
+        BOOLEAN no_auto_firmware;
```

Comment:
I think the naming of these variables `no_auto_...` should be `auto_...`, then inverted value should be stored.

Reply:
Normally, I'd have done so. I'm just following the old code here (no_editor).

Reply:
It is not necessary to follow the naming `no_editor`. Also, `no_editor` should be changed to `editor` or something later. (Of course, changing that in this PR is welcome.)